lib/Support/MSFileSystemBasic.cpp

```diff
@@ -942,6 +947,23 @@ int MSFileSystemForIface::Write(int fd, const void* buffer, unsigned int count)
   return (int)cbWritten;
 }
 
+#ifndef _WIN32
+int MSFileSystemForIface::Open(const char *lpFileName, int flags, mode_t mode) throw() {
+  SetLastError(ERROR_FUNCTION_NOT_CALLED);
```

Comment:
> #define ENOSYS          38      /* Function not implemented */

Add this to WinAdapter.h?

```cpp
#define ERROR_FUNCTION_NOT_CALLED ENOSYS
```

Reply:
I suppose I could, but this entire file is excluded on non-windows platforms using ifdef _WIN32. So such a define would never be expanded.  I added them here merely as placeholders should this ever be revived for non-Windows builds.